assets/sass/patterns/organisms/content-header-journal.scss

```diff
@@ -38,6 +38,30 @@
   }
 }
 
+.content-header__subject_link {
+  text-decoration: none;
+
+  .content-header__subject {
+    @include margin(12, "right");
+    @include padding(5 10 3 10);
+    background-color: #ebf2fa;
+    border-radius: 3px;
+    color: $color-primary;
+    font-weight: bold;
+  }
+}
+
+.content-header__subject_list_item > .content-header__subject {
+  @include font-size-and-vertical-height(12);
+  @include margin(12, "right");
+  background-color: #edeff4;
```

Comment:
Can we check these colours with the design and Chris please as they do not appear in variables.scss